Makefile

```diff
@@ -51,4 +60,33 @@ testdata/firecracker:
 testdata/root-drive.img:
 	$(curl) -o $@ https://s3.amazonaws.com/spec.ccfc.min/img/hello/fsfiles/hello-rootfs.ext4
 
+tools/firecracker-builder-stamp: tools/docker/Dockerfile
+	docker build \
+		-t localhost/$(FIRECRACKER_BUILDER_NAME):$(DOCKER_IMAGE_TAG) \
```

Comment:
This `DOCKER_IMAGE_TAG` variable must be referenced and used from BuildKite to run multiple PR builds correctly.

Reply:
I'm not following. How does this prevent multiple builds?

Reply:
Docker image `localhost/firecracker-builder:latest` is shared by multiple builds and not immutable. We should have something like https://github.com/firecracker-microvm/firecracker-containerd/blob/master/.buildkite/pipeline.yml#L22

Reply:
Done